templates/module/rest-api/router-template.ts

```diff
@@ -0,0 +1,24 @@
+import { accessAuthMiddleware } from '../../access-token';
+import { ApplicationRouter } from '../../application';
+
+import { $entityNameController } from './$moduleName-controller';
+
+export default class $entityNameRouter extends ApplicationRouter {
+  configure(): void {
+    const { router } = this;
+    const ctrl = new $entityNameController();
+
+    router.use(accessAuthMiddleware);
+
+    const baseUrl='$baseUrl'
```

Comment:
I don't think this is needed as the base url is usually be similar to the module name(similar to the existing server url). But if you think this has some use-case, can we mover this to server class. As, it seems like the baseUrl is added to all the routes.